togetherly/lib/views/widgets/chore_details_dialog.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:flutter/material.dart';
+import 'package:togetherly/models/chore.dart';
+import 'package:togetherly/themes.dart';
+
+class ChoreDetailsDialog extends StatelessWidget {
+  const ChoreDetailsDialog({super.key, required this.chore});
+  final Chore chore;
+
+  Widget buildTextBox(BuildContext context, String title, String description) {
+    return Container(
+      margin: const EdgeInsets.only(bottom: 5),
+      width: MediaQuery.of(context).size.width,
+      decoration: BoxDecoration(
+          color: AppColors.brandLightGray,
+          borderRadius: BorderRadius.circular(5)),
+      child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              title,
+              style: AppTextStyles.brandBody,
+            ),
+            Text(
+              description,
+              style: AppTextStyles.brandBodySmall
+                  .copyWith(fontStyle: FontStyle.italic),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget buildIconText(IconData icon, String title) {
+    return Row(
+      mainAxisSize: MainAxisSize.min,
+      mainAxisAlignment: MainAxisAlignment.end,
+      children: <Widget>[
+        Icon(
+          icon,
+          color: AppColors.brandGold,
+          size: 24,
+        ),
+        Text(
+          title,
+          style: AppTextStyles.brandAccentLarge,
+        ),
+      ],
+    );
+  }
+
+  String prettyDate(String date) {
+    List<String> dateTimeArr = date.split(' ');
+    List<String> dateArr = dateTimeArr.elementAt(0).split('-');
+    return '${dateArr.elementAt(1)}/${dateArr.elementAt(2)}/${dateArr.elementAt(0)}';
```

Comment:
This could be more readable if you instead took a DateTime object and pulled out `date.month`, `date.day`, and `date.year` into variables, rather than splitting the string representation.